schemas/entity/assessment.schema.tpl.json

```diff
@@ -0,0 +1,21 @@
+{
+  "_extends": "/publications/schemas/creativeWork.schema.tpl.json",
+  "_type": "https://openminds.ebrains.eu/specimenPrep/AssessmentInstrument",
```

Comment:
type mismatches type of filename and of reference in the other schema. Plus: I don't think this is a separate schema. Questionnaires or behavioral assessments are captured in behavioral protocols.